code/game/turfs/closed/minerals.dm

```diff
@@ -51,10 +51,18 @@
 		return
 
 	var/mob/living/bumping = bumped_atom
+	if(!ISADVANCEDTOOLUSER(miner)) // Unadvanced tool users can't mine anyway (this is a lie). This just prevents message spam from attackby()
+		return
+
 	var/obj/item/held_item = bumping.get_active_held_item()
+	if(iscyborg(miner))
+		var/mob/living/silicon/robot/robot = miner
+		held_item = robot.module_active
```

Comment:
Is this not the automatic behavior of robot.get_active_held_item()? :(

Reply:
oh shit wait it is, I thought that was stupid